certified-connectors/Africa's Talking Payments/apiDefinition.swagger.json

```diff
@@ -1,4 +1,4 @@
-{
+{{
```

Comment:
Hi @jokano12, can you check why is there a change in the definition for a different connector?